packages/ui/cypress/e2e/login.cy.ts

```diff
@@ -0,0 +1,24 @@
+import { injectedAccounts } from '../fixtures/injectedAccounts'
+import { localHost } from '../fixtures/landingData'
+
+describe('login-accounts', () => {
```

Comment:
```suggestion
describe('Connect Account', () => {
```
This is just what appears in Cypress / any reporting. I usually write it like this

Reply:
sure